### PR TITLE
Feature/ edit user

### PR DESCRIPTION
**Pull request by AnnaEmson**
### What

New auth work - add ability to update user details and activate or suspend user. 

Screenshot 2022-02-24 at 17 33 09
Screenshot 2022-02-24 at 17 32 58

### How to review

Describe the steps required to test the changes.

### Who can review

Describe who worked on the changes, so that other people can review.

### CODE REVIEW

File: src/app/views/users/edit/EditUser.jsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+import validate from "./validate";
+import FormValidationErrors from "../../../components/form-validation-errors";
+import BackButton from "../../../components/back-button";
+import FormFooter from "../../../components/form-footer";
+import SelectedItemList from "../../../components/selected-items/SelectedItemList";
+import Loader from "../../../components/loader/index";
+import RadioGroup from "../../../components/radio-buttons/RadioGroup";
+import TextArea from "../../../components/text-area/TextArea";
+
+const options = [
```
Comment (dandwal) [inline src/app/views/users/edit/EditUser.jsx:19]: Not a required change but a note, the name `options` is a bit generic, another developer who may need to maintain this in the future will not easily know what 'options' these relate to.

File: src/app/views/users/groups/AddGroupsToUser.jsx [hunk 2]
```diff
@@ -15,18 +17,34 @@ const notification = {
     autoDismiss: 5000,
 };
 
+const leaveNotification = {
+    type: "neutral",
+    message: "dddddd.",
```
Comment (dandwal) [inline src/app/views/users/groups/AddGroupsToUser.jsx:22]: I think the message needs updating?

File: src/app/views/users/UsersList.jsx [hunk 3]
```diff
@@ -1,180 +1,81 @@
...
     }, []);
 
-    const handleSearchInput = event => {
-        const searchTerm = event.target.value.toLowerCase();
-        const filteredUsers = props.users.filter(
-            user => user.title.toLowerCase().search(searchTerm) !== -1 || user.email.toLowerCase().search(searchTerm) !== -1
+    const getFilteredUsers = useCallback(() => {
+        const str = search.value.toLowerCase();
+        return users.filter(
+            user => user.forename.toLowerCase().includes(str) || user.lastname.toLowerCase().includes(str) || user.email.toLowerCase().includes(str)
```
[the remark this answers is not in the document]
Comment (AnnaEmson) [reply src/app/views/users/UsersList.jsx:24]: good point will do

**Review by dandwal — DISMISSED** [review]
LGTM, have had a demo of the code in play too.

**Review by dandwal — APPROVED** [review]

**Pull request merged**